superset-frontend/src/dashboard/components/nativeFilters/FilterConfigForm.tsx

```diff
@@ -148,57 +172,59 @@ export const FilterConfigForm: React.FC<FilterConfigFormProps> = ({
           filterId={filterId}
           datasetId={dataset?.value}
         />
-      </Form.Item>
-      <Form.Item
+      </StyledFormItem>
+
+      <StyledFormItem
         name={['filters', filterId, 'defaultValue']}
-        label={t('Default Value')}
+        label={<StyledLabel>{t('Default Value')}</StyledLabel>}
         initialValue={filterToEdit?.defaultValue}
       >
         <Input />
-      </Form.Item>
-      <Form.Item
+      </StyledFormItem>
+      <StyledFormItem
         name={['filters', filterId, 'parentFilter']}
-        label={t('Parent Filter')}
+        label={<StyledLabel>{t('Parent Filter')}</StyledLabel>}
         initialValue={parentFilterOptions.find(
           ({ value }) => value === filterToEdit?.cascadeParentIds[0],
         )}
       >
         <Select options={parentFilterOptions} isClearable />
-      </Form.Item>
-      <Form.Item
+      </StyledFormItem>
```

Comment:
I think it could be helpful to change the default placeholder from `Select...` to `None` or siilar to more explicitly communicate that defining a parent is not required.

Reply:
Good point!